app/views/placements/show.html.erb

```diff
@@ -26,18 +28,19 @@
     </div>
 </div>
 
-<div class='card bg-light col-4 rounded mb-5'>
+<div class='shadow-sm card bg-light col-4 rounded mb-5'>
     <div class='card-body'>
         <h4>Other</h4>
         <ul class='list-group'>
             <li class='list-group-item'>Gender: <%= @placement.gender %></li>
             <li class='list-group-item'>Beds: <%= @placement.beds %></li>
         </ul>
         <br>
-        <%= link_to 'Edit', edit_placement_path(@placement.id), class: 'btn btn-primary mt-2' %> 
-        <br>
-        <%= link_to 'Delete', placement_path(@placement.id), method: :delete, class: 'btn btn-danger mt-2' %>
-
+        <% if current_user && current_user.admin? %>
+            <%= link_to 'Edit', edit_placement_path(@placement.id), class: 'btn btn-primary' %> 
+        
+            <%= link_to 'Delete Placement', placement_path(@placement.id), method: :delete, class: 'btn btn-danger ml-2' %>
```

Comment:
You don't need to specify `.id` here; Rails is smart enough to figure that out.

Reply:
Done.